USER_GUIDE.md

```diff
@@ -84,7 +89,7 @@ There are multiple low level transports which `OpenSearchClient` could be config
 import org.apache.http.HttpHost;
 
 final HttpHost[] hosts = new HttpHost[] {
-    new HttpHost("localhost", 9200, "http")
+    new HttpHost("http", "localhost", 9200)
```

Comment:
@maltehedderich this change for `2.x` we have to revert, `org.apache.http.HttpHost` is using different constructor